### PR TITLE
Remove the Join the Family heading in the student ambassador page mobile view

### LINKED ISSUE #1368: Remove the "Join the Family" heading in the student ambassador page mobile view
**Is your feature request related to a problem? Please describe.**
- There's a heading on the student ambassador page only in the mobile view. We don't need that heading.

**Describe the solution you'd like**
- Remove the `Join the Family` heading

image

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1368 

## Goals

Removing the Join the Family heading in the student ambassador page mobile view

## Approach

Removed the Join the Family heading in the student ambassador page mobile view

### Screenshots

image

  
### Preview Link

https://pr-1369-sef-site.surge.sh/student-ambassador.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: student-ambassador.html [hunk 1]
```diff
@@ -53,7 +53,6 @@ <h1>STUDENT <b>AMBASSADOR</b></h1>
 
             <!--            For mobile view-->
             <div class="d-block d-md-none mt-5 py-4">
-                <p style="line-height: 1.0;"> <h1 style='font-size:65px;'> <span style='font-weight:bold;'>Join the</span> Family</h1> </p>
                 <p > <h2>STUDENT <b>AMBASSADOR</b></h2>
                 </p>
```
Comment (anjula-sack) [inline student-ambassador.html]: We don't need this `<p>` tag right?

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: student-ambassador.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply student-ambassador.html]: Yes, ok then I will remove it @anjula-sack

Comment (YoshithaRathnayake) [reply student-ambassador.html]: Done @anjula-sack

Comment (YoshithaRathnayake) [reply student-ambassador.html]: Can you merge it? @anjula-sack

**Review by anjula-sack — APPROVED** [review]
Thank you! @YoshithaRathnayake

**Pull request merged**